src/App.js

```diff
@@ -9,9 +9,24 @@ import { UserInfo } from "./pages/UserInfo/UserInfo";
 import { Routes, Route } from "react-router-dom";
 import { RequiresAuth } from "./RequiresAuth";
 import { NotFound } from "./pages/NotFound/NotFound";
+import { ProductPage } from "../src/pages/ProductPage/ProductPage";
+import { ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+const notify = () => {
+  toast("🦄 Wow so easy!", {
```

Comment:
if you added it for testing purpose, and if not needed for future use, then you can remove this toast from here, and also the button at line no. 29 in this file.

PS: all your toasts used at other places are working as expected.

Reply:
Thanks for pointing, removed it!


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable